addons/binding/org.openhab.binding.dsmr/src/main/java/org/openhab/binding/dsmr/internal/discovery/DSMRMeterDiscoveryService.java

```diff
@@ -160,12 +160,12 @@ private void validateConfiguredMeters(List<Thing> things, Set<DSMRMeterType> con
      * @param unconfiguredMeters The list of meters that were detected, but not configured
      */
     protected void reportConfigurationValidationResults(List<DSMRMeterType> invalidConfigured,
-            List<DSMRMeterType> unconfiguredMeters) {
+        List<DSMRMeterType> unconfiguredMeters) {
         logger.info(
-                "Possible incorrect meters configured. These are configured: {}."
-                        + "But the following unconfigured meters are found in the data received from the meter: {}",
-                invalidConfigured.stream().map(m -> m.name()).collect(Collectors.joining(", ")),
-                unconfiguredMeters.stream().map(m -> m.name()).collect(Collectors.joining(", ")));
+            "Possible incorrect meters configured. These are configured: {}."
+                + "But the following unconfigured meters are found in the data received from the meter: {}",
```

Comment:
Can we avoid using String concatenation?

Reply:
@cweitkamp the discovery works now for my meter. So I'm not able to verify the fixed logging of the raw data.

Reply:
@BoBiene Thanks. Good to heart. Regarding the logging I trust in @Hilbrand .